desktop/apps/auctions2/components/auctions_block/AuctionsBlock.js

```diff
@@ -0,0 +1,84 @@
+import moment from 'moment'
+import PropTypes from 'prop-types'
+import React from 'react'
+
+function liveDate(auction) {
+  if (moment(auction.registration_ends_at) > moment()) {
+    return formattedDate(auction.registration_ends_at, false, true)
+  } else if (moment(auction.live_start_at) > moment()) {
+    return formattedDate(auction.live_start_at)
+  } else if (moment(auction.end_at) > moment()) {
+    return 'In Progress'
+  }
+}
```

Comment:
One thing that may fail here is that most live integration sales have no `end_at`, so we might be able to just use an `else` there or check if `moment() > auction.live_start_at`.